grafana.tf

```diff
@@ -1,21 +1,14 @@
 locals {
-  grafana_dashboard_create            = var.grafana_dashboard_create && lookup(module.this.tags, "Type", null) != "scheduled" ? 1 : 0
-  grafana_elasticsearch_index_pattern = var.grafana_elasticsearch_index_pattern != "" ? var.grafana_elasticsearch_index_pattern : "logs-${module.this.namespace}-${module.this.stage}-${module.this.name}"
-}
-
-data "gosoline_application_dashboard_definition" "main" {
-  count       = local.grafana_dashboard_create
-  project     = module.this.namespace
-  environment = module.this.environment
-  family      = module.this.namespace
-  application = module.this.name
-  containers  = var.containers
-  group       = module.this.stage
+  grafana_dashboard_create            = var.grafana_dashboard_enabled && lookup(module.this.tags, "Type", null) != "scheduled" ? 1 : 0
+  grafana_elasticsearch_index_pattern = var.grafana_elasticsearch_index_pattern != "" ? var.grafana_elasticsearch_index_pattern : "logs-${module.ecs_label.id}"
+  containers                          = var.containers != null ? var.containers : [module.ecs_label.id, "log_router"]
```

Comment:
one of these must be wrong. If the ecs_label contains more than group and application, then it's definitely not the container_name. If the ecs_label does not contain env, (family?), group and application then it's definitely not the grafana elasticsearch index pattern.